docs/tbdev_getting_started.ipynb

```diff
@@ -12,7 +12,7 @@
     },
```

Comment:
<p>instead of "provides a managed TB experience" can we rephrase as:</p><p> </p><p>TensorBoard.dev is a free, public TensorBoard service that enables you upload and share your ML experiment results with everyone.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
done